File: python/sample-authentication-code/sendFirstRequest.py
import requests

'''
This snippet is used to send a request to our servers. 
Combine it with our getAccessToken to get started
'''

def send_first_request(url, access_token):
    headers = {
    "Content-Type": "application/json",
    "Accept": "application/json",
    "Authorization": "Bearer " + access_token    
    }
    response = requests.get(url, headers=headers)
    return response.json()

File: js/sample-authentication-code/sendFirstRequest.js
/**
 * This script gives you the code for sending a request to one of our APIs
 */

const axios = require("axios");

const url = "https://api-mock.payments.jpmorgan.com/tsapi/v1/payments";

const sendFirstRequest = async (body, accessToken) => {
  try {
    const response = await axios.post(url, body, {
      headers: {
        "Content-Type": "application/json",
        Accept: "application/json",
        Authorization: `Bearer ${accessToken}`,
      },
    });
    console.log(response.data);
  } catch (error) {
    console.error(error);
  }
};
